bundles/core/src/test/resources/image/v3/exporter-image38-two-smart-sizes.json

```diff
@@ -2,8 +2,7 @@
   "id": "image-cfa1cb817f",
   "alt": "Adobe Systems Logo and Wordmark in PNG format",
   "title": "Adobe Systems Logo and Wordmark",
-  "srcset":"https://s7d9.scene7.com/is/image/dmtestcompany/Adobe_Systems_logo_and_wordmark_DM.600?ts=1490005239000&dpr=off 600w,https://s7d9.scene7.com/is/image/dmtestcompany/Adobe_Systems_logo_and_wordmark_DM.800?ts=1490005239000&dpr=off 800w",
-  "src": "https://s7d9.scene7.com/is/image/dmtestcompany/Adobe_Systems_logo_and_wordmark_DM?ts=1490005239000&dpr=off",
+  "src": "https://s7d9.scene7.com/is/image/dmtestcompany/Adobe_Systems_logo_and_wordmark_DM?ts=1490005239000&dpr=on,{dpr}",
```

Comment:
Shouldn't the template also be part of the exported JSON? How would a clientside/SPA implementation rebuild the component?

Reply:
The current exported value for the `srcset` is wrong for dm images which have a `smartCropRendition=SmartCrop:Auto` and should be considered as a 🐛. In the other PR I tried to generate the array on server side but this approach was not successful because:
1. The `DynamicMediaRenditionProvider` is not reliable. Sometimes the renditions got not created in the JCR.
2. The device pixel ratio can only be detected on client side.

For a clientide/SPA implementation it should be easy to adapt the existing client side solution of this PR.